Watchman.Discord/Integration/DevscordFramework/CustomCommandsLoader.cs

```diff
@@ -39,9 +39,15 @@ public async Task InitDefaultCustomCommands()
             {
                 customCommands.AddRange(new List<AddCustomCommandsCommand>
                 {
-                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.AddRoleCommand", @"-add\s*role\s*(?<Roles>[\w\W\s\""]*)", serverId),
-                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.RemoveRoleCommand", @"-remove\s*role\s*(?<Roles>[\w\W\s\""]*)", serverId),
-                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Administration.BotCommands.SetRoleCommand", @"-set\s*role\s*(?<Roles>[\w\W\s\""]*)\s*-((?<Safe>safe)|(?<Unsafe>unsafe))", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.AddRoleCommand", @"-add\s*role\s*(?<Roles>.*)", serverId),
```

Comment:
@gagyn sprawdzałeś dokładnie te regexy, czy lepiej dokładnie to przejrzeć?

Reply:
chyba tylko te od roli sprawdzałem

Reply:
no to lepiej sprawdzić wszystkie, bo są tu takie rzeczy jak `(?<Time>.*)` 

przez co komenda przejdzie a później sie wywali i będzie narzekanie jakie to wszystko jest złe i nie działa

Reply:
no właśnie tak ma być. Dzięki temu zamiast braku reakcji ze strony mamy komunikat, że albo są niepoprawne argumenty albo, że jest nie wystarczająca liczba argumentów.

Wiem, że w przyszłości będą helpy do komend, ale niech na razie tak będzie, poza tym do tych helpów też może by to było potrzebne.